bin/backrest-init.js

```diff
@@ -11,11 +11,13 @@ program
       name
     }
 
+    let projectPath = `${process.cwd()}/${name}`
+    fs.mkdirSync(projectPath)
```

Comment:
This line may seem redundant, but if the folder isn't already made prior to us going into the `_.each`, it throws an error when creating the `/app` folder